hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -6,10 +6,16 @@
   OVS installation directory. It is the path argument when using Install-OVS.ps1. The default path is "C:\openvswitch".
   .PARAMETER RenewIPConfig
   Renew the ipconfig on the host. The default value is $false.
+  .PARAMETER RemoveOVS
+  Remove ovsdb-server and ovs-vswitchd services on the host. The default value is $false. If this argument is set as
+  true, this script would remove the two Windows services of ovsdb-server and ovs-vswitchd from the host. Otherwise,
+  these services are thought to be supposed to be running on the host, so the script would try to recover them if their
```

Comment:
s/these services are thought to be supposed to be running on the host/we consider that these services are supposed to be running on the host